code/game/objects/items/defibrillator.dm

```diff
@@ -379,5 +385,6 @@
 	if(istype(user) && istype(target))
 		defibrillate(target, user)
 
-/obj/item/defibrillator/gloves/update_icon_state()
-	return //The parent has some behaviour we don't want
+/obj/item/defibrillator/gloves/update_icon()
+	SHOULD_CALL_PARENT(FALSE)
```

Comment:
Absolutely not. Once you figure out the proper way to handle appearance for defib, you should have a better way to do this.